src/archive/src/main.rs

```diff
@@ -229,8 +246,11 @@ impl BoundedStorable for AnchorIndexKey {
     }
 }
 
+/// This method is kept for legacy compatibility and easier testability of the archive.
```

Comment:
But I'm guessing we'll remove it eventually, right? If we're happy with the poll we have no reason to go back to push